src/constant.js

```diff
@@ -40,8 +42,77 @@ export const HideFooter = [
   PATHS.PROFILE,
   PATHS.MENTOR,
   PATHS.PRIVACY_POLICY,
+  PATHS.VOLUNTEER_FORM,
 ];
 
+const month = {
+  "01": "Jan",
+  "02": "Feb",
+  "03": "Mar",
+  "04": "Apr",
+  "05": "May",
+  "06": "Jun",
+  "07": "Jul",
+  "08": "Aug",
+  "09": "Sep",
+  10: "Oct",
+  11: "Nov",
+  12: "Dec",
+};
+export const lang = {
+  en: "English",
+  hi: "Hindi",
+  mr: "Marathi",
+  ta: "Tamil",
+  te: "Telugu",
+};
+export const dateTimeFormat = (date) => {
+  const datePart = date?.split("T")[0].split("-").reverse();
+  const TimePart = date?.split("T")[1].split(":");
+  const finalDate = `${datePart[0]} ${month[datePart[1]]}, ${datePart[2]} `;
+  const finalTime = `${TimePart[0]} : ${TimePart[1]}`;
+  return { finalTime, finalDate };
+};
+
+export const TimeLeft = (date) => {
```

Comment:
A constants file is usually used for data constants such as numbers, or strings.  Functions like `dateTimeFormat` and `TimeLeft` should probably be put in `utils.js`.  But it'd probably be better to organize these functions in a utils folder as is done here: https://github.com/google/blockly/tree/master/core/utils.  So here, we could use `date.js` to indicate date-related functionality.

Reply:
Sure, will do that.